### PR TITLE
Refactor logging

### PR DESCRIPTION
**Pull request by FalseR20**

### CODE REVIEW

**Review by AlexandrZagorskiy — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -45,32 +45,32 @@ keynodes.resolve("some_node", None)  # The same logic as keynodes.get("some_node
...
-For ScAgent initialization you write event class and event type.
+#### Initialisation
 
-For ScAgentClassic you write name of action class and event type (default value is ScEventType.ADD_OUTGOING_EDGE).
-Event class here is `question_initiated`. There is also method to confirm action class.
+For ScAgent initialisation you write event class and event type.
+
+For ScAgentClassic initialisation you write name of action class (it isn't event class) and args of ScAgent.
+Default `event_class` is `question_initiated`, event_type is `ScEventType.ADD_OUTGOING_EDGE`
+There is also method to confirm action class.
```
[the remark this answers is not in the document]
Comment (FalseR20) [reply README.md:73]: There is it usage [sum_agent_classic.py](docs/examples/sum_agent_classic.py)

File: README.md [hunk 2]
```diff
@@ -45,32 +45,32 @@ keynodes.resolve("some_node", None)  # The same logic as keynodes.get("some_node
...
     def on_event(self, class_node: ScAddr, edge: ScAddr, action_node: ScAddr) -> ScResult:
-        self._logger.info("Agent's called")
         if not self._confirm_action_class(action_node):  # exclusive method for classic agent
             return ScResult.SKIP
-        self._logger.info("Agent's confirmed")
         ...
 ```
 
-For ScAgent initialization you write event class and event type.
+#### Initialisation
```
[the remark this answers is not in the document]
Comment (FalseR20) [reply README.md]: Done

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (FalseR20) [reply README.md]: Done

[the remark this answers is not in the document]
Comment (FalseR20) [reply README.md]: I removed this line

**Review by AlexandrZagorskiy — APPROVED** [review]

**Pull request merged**